scripts/profiler/model_creator.py

```diff
@@ -140,11 +143,7 @@ def transmit_all_events_descriptions(self):
             event_types_dict = dict((k, v.serialize())
                     for k, v in self.processed_events.registered_events_types.items())
             json_et_string = json.dumps(event_types_dict)
-            try:
-                self.out_stream.send_desc(json_et_string.encode())
-            except StreamError as err:
-                self.logger.error("Error: {}. Unable to send data".format(err))
-                sys.exit()
+            self.out_stream.send_desc(json_et_string.encode())
```

Comment:
Maybe we could leave the try catch here. I think we may not make assumptions related to the underlying implementation and if anything goes wrong, we could simply provide an error log and terminate gracefully.

Reply:
That would also let us to properly disconnect/unitialize something if needed (e.g. RTT)